source/jQuery.outerHTML.js

```diff
@@ -0,0 +1,81 @@
+(function ($) {
+    $.fn.extend({
+        merge: function (array) {
+            /* You need this kind of function to be able to
+            * chain jQuery-objects in an empty jQuery, because
+            * append requires connection to the DOM (except you
+            * use document.createDocumentFragmnet() or instantiated
+            * jQuery properly with a any '<tagname/>') and add 
+            * returns a *new* object, instead to reuse itself.
+            */
+            return jQuery.merge(this, array);
```

Comment:
Would checking for uniqueness be beneficial here? Maybe at least have a flag that make a unique set of elements an option: `$([]).merge([], true);` where true would keep only unique elements.

Reply:
Sorry, I am pretty busy. In my opinion, I would not implement it, because the jQuery.merge does not implement it too. The user can have it in mind by hisself by the various of functions that exists already by jQuery or Lodash.